common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/TestUtils.java

```diff
@@ -172,4 +170,54 @@ public static void assertIsSerializable(@NotNull Serializable object) {
       }
     }
   }
+
+
+  /**
+   * Asserts that the given {@link ThrowingRunnable} throws an exception of type {@code
+   * expectedThrowable} when executed.
+   *
+   * @param expectedThrowable the class of the exception that should be thrown
+   * @param runnable the {@link ThrowingRunnable} that should throw
+   * @param <T> the type of exception that should be thrown
+   * @return the thrown exception of type {@code expectedThrowable}
+   * @throws AssertionError if the given {@link ThrowingRunnable} throws a different type of
+   *     exception or does not throw at all
+   */
+  public static <T extends Throwable> T expectThrows(
```

Comment:
AFAIK `assertThrows` is only provided in JUnit 5. The recommended way in JUnit 4 is to use either try/catch or `ExpectedException`. The try/catch idiom is lengthy and makes tests harder to read (hence why JUnit 5 introduces `assertThrows`) and `ExpectedException` will not execute statements beyond the one that immediately follows the expectation setup: https://github.com/junit-team/junit4/wiki/Exception-testing

Bringing an `expectThrows` method in now prevents clashing with the future `assertThrows`, but follows the same pattern.